app/src/org/commcare/models/database/app/AppDatabaseUpgrader.java

```diff
@@ -246,6 +259,12 @@ private boolean upgradeEightNine(SQLiteDatabase db) {
         return success;
     }
 
+    private boolean upgradeNineTen(SQLiteDatabase db) {
+        SqlStorage.wipeTable(db, UPGRADE_RESOURCE_TABLE_NAME);
```

Comment:
I can't think of a scenario where this wouldn't just fail out anyway, but technically these should also be in a transaction, right?